packages/loader/lib/main.js

```diff
@@ -26,12 +25,13 @@ var mainContext = this;
     };
 
     requirejs = require = requireModule = function(name) {
-      var s = seen[name];
+      var exports = seen[name];
 
-      if (s !== undefined) { return seen[name]; }
-      if (s === UNDEFINED) { return undefined;  }
+      if (exports !== undefined) {
```

Comment:
I did remove support for the module callbacks that have a return value, but I verified that none of our modules were doing that anyways.